homeassistant/components/camera/uvc.py

```diff
@@ -127,11 +128,15 @@ def _login(self):
         else:
             client_cls = uvc_camera.UVCCameraClient
 
+        username = caminfo['username']
```

Comment:
Why not follow the same pattern of password ? (self._password) retrieving the default value from the config file

Reply:
I don't know enough about the UVC client, but from what I've seen the client should normally return a username. Mine is not for some reason since it is working for others, I assume the username is set for many people. You may be able to change it to the default but I don't know if it'll break for others. The way I coded it should prevent any existing that are working from starting to fail after this change.

Reply:
You should keep your test of camino['username'], my point is getting the username from the config file and not hardcoding it.